src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -695,8 +695,13 @@ TritonInferInput::Create(
   TritonInferInput* local_infer_input = new TritonInferInput(name, datatype);
 
   tc::InferInput* triton_infer_input;
-  RETURN_IF_TRITON_ERROR(
-      tc::InferInput::Create(&triton_infer_input, name, dims, datatype));
+  bool binary_input_data{true};
+  const char* binary_input_data_env_var{std::getenv("PA_BINARY_INPUT_DATA")};
+  if (binary_input_data_env_var != nullptr) {
+    binary_input_data = std::atoi(binary_input_data_env_var);
```

Comment:
So the feature is controlled by env variables? Do we need throw/catch in case it isn't an integer?  Also, it looks like GRPC Client has a function GetEnvironmentVariableOrDefault that could be useful. Should that be shared so we have common code?

Reply:
This is going into a dev branch. A future story will use a PA command line option.

Reply:
And the PA command line will communicate to the clients via env variables? Or will all of the ENV code be ripped out?

Reply:
Environment variables will be ripped out. It will just be a c++ variable propagated down to the client. I use environment variable here so I can test and so if anyone is using dev branch there's a way to access feature.